src/slic3r/GUI/BitmapCache.cpp

```diff
@@ -322,7 +322,7 @@ wxBitmap* BitmapCache::load_svg(const std::string &bitmap_name, unsigned target_
 
     // map of color replaces
     std::map<std::string, std::string> replaces;
-    replaces["\"#00AE42\""] = "\"#009688\"";
+    replaces["\"#009688\""] = "\"#009688\"";
```

Comment:
This shouldn't change.
I use to replace SVG files' color programmatically.

Reply:
yes  sorry !

Reply:
Not at all.
I appreciate it!